mmpose/datasets/datasets/body/exlpose_dataset.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from mmpose.registry import DATASETS
+from ..base import BaseCocoStyleDataset
+
+
+@DATASETS.register_module()
+class ExlposeDataset(BaseCocoStyleDataset):
+    """Exlpose dataset for pose estimation.
+
+    "Human Pose Estimation in Extremely Low-Light Conditions",
+    arXiv'2023.
+    More details can be found in the `paper
+    <https://arxiv.org/abs/2303.15410>`__
+
+    AIC keypoints:
```

Comment:
Please verify if the description provided in the docstring aligns with the dataset